src/commands/create-manifest/__test__/create-manifest.test.ts

```diff
@@ -87,12 +87,12 @@ describe('createManifest', () => {
     ]);
   });
   describe('validatePaths', () => {
-    it('Should throw error for Missmatch Paths', () => {
+    it('Should throw error for Mismatched Paths', () => {
       assert.throws(() => {
         validatePaths('memory://source/', 'memory://target/sub/test.tiff');
       }, Error);
     });
-    it('Should also throw error for Missmatch Paths', () => {
+    it('Should also throw error for Mismatched Paths', () => {
```

Comment:
Wait, what's the "also" here? If these tests are fundamentally different they should probably be named accordingly.